presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/HiveTableOperations.java

```diff
@@ -82,6 +82,7 @@
             FileInputFormat.class.getName(),
             FileOutputFormat.class.getName());
 
+    private final HiveContext context;
```

Comment:
Let's name this `hiveContext` so that it's not confused with `hdfsContext`, which is also used in this class. (I realize we don't store `hdfsContext` as a field, but seems less confusing to qualify the name)

Move this field below `metastore`, since `metastore` is a service object and the rest are data objects.